<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAllOnlineNormalUsers</class>
 <widget class="QDialog" name="GetAllOnlineNormalUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>201</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Online Normal Users</string>
   </property>
   <widget class="QListWidget" name="listOnlnNormalComm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>331</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>User Details</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection ID</string>
    </property>
    <widget class="QLabel" name="lblOnlnNormalCntnIdComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>User Type</string>
    </property>
    <widget class="QLabel" name="lblOnlnNormalUsrTypeComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Name</string>
    </property>
    <widget class="QLabel" name="lblOnlnNormalNameComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Phone Number</string>
    </property>
    <widget class="QLabel" name="lblOnlnNormalPhComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Address</string>
    </property>
    <widget class="QLabel" name="lblOnlnNormalAddressComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
